pom.xml

```diff
@@ -14,11 +14,34 @@
         <module>dss-server</module>
     </modules>
 
+    <profiles>
+        <profile>
+            <id>jdk11</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+                <jdk>11</jdk>
+            </activation>
+            <properties>
+                <java.version>11</java.version>
+                <access.option>--illegal-access=permit</access.option>
+            </properties>
+        </profile>
+        <profile>
+            <id>jdk8</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+                <jdk>1.8</jdk>
+            </activation>
+            <properties>
+                <java.version>1.8</java.version>
+            </properties>
+        </profile>
+    </profiles>
+
```

Comment:
It is greate idea that add the profiles!

Reply:
If this works well, we also can close #73

Reply:
Right! I hope to resolve both...plz

Reply:
Oh..I think this works on travis. Is it right?